src/components/form/fields/DateField.tsx

```diff
@@ -29,10 +29,18 @@ export function DateField({
     const locale = selectedLocale
     const [validation, setValidation] = useState<ValidationProps>({
         error: false,
+        valid: true,
     })
 
     const { input, meta } = useField<string | undefined>(name)
 
+    useEffect(() => {
+        if (input.value) {
+            const dateWithNoTimestamp = input.value.substring(0, 10)
```

Comment:
Why is this an `useEffect` and not done in `handleChange` below? Is it because we also want to run this on initial value?

I think we also could use `format` and/or `parse` in `useField`:

```
    const { input, meta } = useField<string | undefined>(name, {
        parse: (value) => value && value.substring(0,10),
    })
````
This would keep the full substring for the `value` - but in the formState it would be stripped. Not sure what we want.

Reply:
yes it is because we only want it to work for the initial value (not only, it's ok if we rerun it every time, but handleChange does not run without an event occurring). Tried with format before, but the calendar library gives a validation error because of the format on the value, so users would need to touch the field before pressing save. Not sure if you have other ideas on how to achieve this? (i'm removing the onblur, i don't think that's needed )